docs/user-guide/src/baremetal/guide.md

```diff
@@ -0,0 +1,242 @@
+
+  Create yaml files for each BareMetalHost that will be used. Below is an example.
+
+  ```yaml
+  ---
+  apiVersion: v1
+  kind: Secret
+  metadata:
+    name: <<secret_name_bmh1>>
+  type: Opaque
+  data:
+    username: <<username_bmh1>>
+    password: <<password_bmh1>>
+  ---
+  apiVersion: metal3.io/v1alpha1
+    kind: BareMetalHost
+    metadata:
+      name: <<id_bmh1>>
+    spec:
+      online: true
+      bootMACAddress: <<mac_address_bmh1>>
+      bootMode: legacy
+      bmc:
+        address: <<address_bmh1>> // this depends on the protocol that are mentioned above, they depend on hardware vendor
+        credentialsName: <<secret_name_bmh1>>
+        disableCertificateVerification: true
+  ```
+
+  Apply the manifests.
+
+  ```bash
+  kubectl apply -f ./bmh1.yaml -n metal3
+  ```
+
+  At this point, the BareMetalHosts will go through `registering` and `inspection` phases before they become `available`.
+
+  Wait for all of them to be available. You can check their status with `kubectl get bmh -n metal3`.
+
+  The next step is to create a workload cluster from these BareMetalHosts.
+
+## Create and apply cluster, controlplane and worker template
+
+  ```bash
+  # The URL of the kernel to deploy.
+  export DEPLOY_KERNEL_URL="http://172.22.0.1:6180/images/ironic-python-agent.kernel"
+
+  # The URL of the ramdisk to deploy.
+  export DEPLOY_RAMDISK_URL="http://172.22.0.1:6180/images/ironic-python-agent.initramfs"
+
+  # The URL of the Ironic endpoint.
+  export IRONIC_URL="http://172.22.0.1:6385/v1/"
+
+  # The URL of the Ironic inspector endpoint.
+  export IRONIC_INSPECTOR_URL="http://172.22.0.1:5050/v1/"
+
+  # Do not use a dedicated CA certificate for Ironic API. Any value provided in this variable disables additional CA certificate validation.
+  # To provide a CA certificate, leave this variable unset. If unset, then IRONIC_CA_CERT_B64 must be set.
+  export IRONIC_NO_CA_CERT=true
+
+  # Disables basic authentication for Ironic API. Any value provided in this variable disables authentication.
+  # To enable authentication, leave this variable unset. If unset, then IRONIC_USERNAME and IRONIC_PASSWORD must be set.
+  export IRONIC_NO_BASIC_AUTH=true
+
+  # Disables basic authentication for Ironic inspector API. Any value provided in this variable disables authentication.
+  # To enable authentication, leave this variable unset. If unset, then IRONIC_INSPECTOR_USERNAME and IRONIC_INSPECTOR_PASSWORD must be set.
+  export IRONIC_INSPECTOR_NO_BASIC_AUTH=true
+
+  # Export node image variable and node image hash varibale that we created before. Change name according to what was downlowded from artifactory
+  export IMAGE_RAW_URL=http://172.22.0.1/images/CENTOS_9_NODE_IMAGE_K8S_v1.27.1-raw.img
+  export IMAGE_RAW_CHECKSUM=http://172.22.0.1/images/CENTOS_9_NODE_IMAGE_K8S_v1.27.1-raw.img.sha256sum
+
+  # Generate templates with clusterctl, change control plane and worker count according to the number of bmhs
```

Comment:
Could you wrap the comments so we don't get so long lines? In the web page preview these go outside the code box so it is necessary to scroll to see them.